docs/source/visualisation/experiment_tracking.md

```diff
@@ -251,22 +276,31 @@ confusion_matrix:
   versioned: true
 ```
 
-After running the pipeline with `kedro run`, the plot will be saved and you will be able to see the plot in the experiment tracking panel when you execute `kedro viz`. Clicking on a plot will expand it. When in comparison view, expanding a plot will show all the plots in that view for them to be compared side-by-side.
+After running the pipeline with `kedro run`, the plot is saved and you can see it in the experiment tracking panel when you execute `kedro viz`. Clicking on a plot expands it. When in comparison view, expanding a plot shows all the plots in that view for side-by-side comparison.
 
-![](../meta/images/expand-plot-comparison-view.gif)
+![](../meta/images/experiment-tracking-plots-comparison.png)
+![](../meta/images/experiment-tracking-plots-comparison-expanded.png)
```

Comment:
Can we please add a space in between these images? To the eye they can be mistaken for one single image.

<img width="1086" alt="image" src="https://user-images.githubusercontent.com/16869061/217313982-8912c125-6e89-4f40-91f7-e46a12b7e886.png">

```suggestion
![](../meta/images/experiment-tracking-plots-comparison.png)

![](../meta/images/experiment-tracking-plots-comparison-expanded.png)
```